Makefile

```diff
@@ -458,6 +458,13 @@ create-mgt-cluster: $(CTLPTL) ## Creates kind-dev Management Cluster
 	rm -f $(CAPHV_WORKER_CLUSTER_KUBECONFIG) $(CAPHV_MGT_CLUSTER_KUBECONFIG)
 	./hack/kind-dev.sh
 
+.PHONY: delete-workload-cluster
+delete-workload-cluster: ## Deletes the example workload Kubernetes cluster
+	@test $${CLUSTER_NAME?Please set environment variable}
+	@echo 'Your Hivelocity resources will now be deleted.'
+	kubectl wait --for=delete cluster/$(CLUSTER_NAME) --timeout=15m
```

Comment:
Just noticed `k delete cluster/hvtest-aniruddha` deletes the cluster successfully but `k wait --for=delete cluster/hvtest-aniruddha --timeout=10m` doesn't

Reply:
@aniruddha2000 `k wait ...` does nothing but waiting. You need two lines: the first line deletes the cluster, the second waits until the action of the first line was done. Please update the PR. thank you.